src/CodeStyle/BannedSymbols.txt

```diff
@@ -0,0 +1,8 @@
+P:Microsoft.CodeAnalysis.Project.LanguageServices; Use 'GetExtendedLanguageServices' instead in CodeStyle layer (preprocessor variable 'CODE_STYLE')
```

Comment:
This ensures CodeStyle layer does not access any Options related types from Workspaces layer.